app/leaderboard/page.tsx

```diff
@@ -64,7 +64,7 @@ export default function Page() {
   const [showNoresults, setShowNoresults] = useState(false);
   const [userAddress, setUserAddress] = useState<string>("");
   const isMobile = useMediaQuery("(max-width:768px)");
-  const [ranking, setRanking] = useState<RankingData>({
+  const [ranking, setRanking] = useState<RankingData | undefined>({
```

Comment:
can you avoid adding undefined here? you can add a error page screen which we use in the codebase if the data doesn't come back but i would personally like to avoid this value inside the state variable

Reply:
Hmm okay i can do that):
I think error state has already been handled in that component.
What i can do is to check for response before calling setRank so to please typescript

Reply:
yep